### ISSUE TITLE
Seal of Insight CRITICAL Bug!

### ISSUE THREAD

**Issue by Renehadox** [gamefreedomgit/Maelstrom#4533]
[//]: # (REMBEMBER! Add links to things related to the bug using for example:)
[//]: # (http://wowhead.com/)
[//]: # (cata-twinhead.twinstar.cz)

**Description:**

If memory serves, Seal of Insight should heal on every hit, here it rarely procs and its crucial for Ret Paladin PvP in high mmr arenas where ret is getting tunneled and needs to survive.

WTB Fix for this please.

**How to reproduce:**

**How it should work:**

**Database links:**

https://wowpedia.fandom.com/wiki/Seal_of_Insight

**Comment by Pornoodemon** [comment]
Or maybe  memory still fails?

**Comment by Renehadox** [comment]
> Or maybe memory still fails? If the tooltip indicates that there is a chance of triggering, this is no longer every hit

You got a problem?

It should have a 75% proc chance ppm...

**Comment by bonfax4** [comment]
15 ppm. Should be  @ 90% if you're using a 3.6 weapon.

**Comment by Sketish** [comment]
It should have a chance not to proc on each attack + your source proves nothing

**Comment by Renehadox** [comment]
> It should have a chance not to proc on each attack + your source proves nothing

Like bonfax4 said " 15 ppm. Should be @ 90% if you're using a 3.6 weapon. "

the faster the weapon the more it will proc, it should proc on every hit, im pretty much sure cause on other servers it worked like that.

@bonfax4 do you mind to provide source of the proc chance so our developer can see ?!

**Comment by bonfax4** [comment]
https://web.archive.org/web/20121117111830/https://www.wowhead.com/spell=20165#comments

> >The proc rate on the heal is 15ppm. This means that with a 2.6speed weapon it's heal proc is 65%, and with a 2.8 it procs 70% of the time. A 3.8second weapon will proc 95% of the time.
> 
> Haste and reckoning do not change the proc rate.

https://www.mmo-champion.com/threads/961212-Tank-Seal/page2

> SoI has a 15 ppm procc chance. 

Btw  faster weapons have a lower the proc chance overall because weapon abilities procs are also based on weapon speed as well.

**Comment by SexyPancake** [comment]
> https://web.archive.org/web/20121117111830/https://www.wowhead.com/spell=20165#comments
> 
> > > The proc rate on the heal is 15ppm. This means that with a 2.6speed weapon it's heal proc is 65%, and with a 2.8 it procs 70% of the time. A 3.8second weapon will proc 95% of the time.
> > 
> > 
> > Haste and reckoning do not change the proc rate.
> 
> https://www.mmo-champion.com/threads/961212-Tank-Seal/page2
> 
> > SoI has a 15 ppm procc chance.
> 
> Btw faster weapons have a lower the proc overall because weapon abilities procs are also based on weapon speed as well.
> 
> Weapon speed * ppm / 60 * 100 to know the %, if you're testing.

Your 1st link have contradicting values as another post is affirming 8.5ppm with the same weapon attack speed.

> I went and hit a target dummy (autoattack only) for roughly 10min. Sadly I was at full health and I found out only later that recount doesn't record overhealing, so here's what I did without real proc counts:
> 
> I had 133 normal melee swings and 10 crits, so a total of 143 swings.
> I got 85k mana back and each proc gave me roughly 1k, so that's 85 procs.
> 
> That is 8.5 PPM and a proc chance of 60%. I had a 2.6 swing timer.
> Obviously, in a real fight, you'd get much more procs, because of special attacks and [(Spell #20179)](https://web.archive.org/web/20121117111830mp_/http://www.wowhead.com/spell=20179);
> I did 580 hps on myself, only from that (I was tanking), on [Coren Direbrew](https://web.archive.org/web/20121117111830mp_/http://www.wowhead.com/npc=23872).

**Comment by bonfax4** [comment]
@SexyPancake Check his math... he just calculated his ppm wrong. 85/135 * 100 = 63% roughly.  

15 ppm with a 2.6 weapon is roughly 65% proc.

8.5 ppm with a 2,6 weapon is rouhly 36% proc.

**Comment by Renehadox** [comment]
@bonfax4 

So seal of insight works properly on this server from what i see from your comments?

**Comment by bonfax4** [comment]
> @bonfax4
> 
> So seal of insight works properly on this server from what i see from your comments?

No idea did not test at all. Just brought the requested info.

**Comment by Renehadox** [comment]
> > @bonfax4
> > So seal of insight works properly on this server from what i see from your comments?
> 
> No idea did not test at all. Just brought the requested info.
> 
> Do like a 1k swing test and add to your post

1000 white hits you mean ? or using melee abilities also?

**Comment by bonfax4** [comment]
White hits yeah. Not sure if details tracks seal procs so you might have to do what the guy on wowhead did. ( mana gained / mana gain per hit ) to get the amount.

**Comment by Renehadox** [comment]
> White hits yeah (you could do specials too but white hits u can just afk ). Not sure if details tracks seal procs so you might have to do what the guy on wowhead did. ( total mana gained / mana gain per hit ) to get the amount.

I am using Recount, and i will track the amount of procs from " Overhealing Done "

**Comment by bonfax4** [comment]
Did a quick test with a 4.0 weapon which should give 100% chance with 15 ppm and it did not proc a couple times so smthign is def wrong.

![5919d86e0673768f405a1ce5b1a0c271](https://github.com/gamefreedomgit/Maelstrom/assets/16911730/06aacbb5-132d-41ae-82fc-f850f92955c9) 
![5c1d57e600a0159050b5f3141a511cb7](https://github.com/gamefreedomgit/Maelstrom/assets/16911730/2786c132-95fc-45ab-9828-6144af5c6364)

**Comment by Renehadox** [comment]
> Did a quick test with a 4.0 weapon which should give 100% chance with 15 ppm and it did not proc a couple times so smthign is def wrong.
> 
> ![5919d86e0673768f405a1ce5b1a0c271](https://user-images.githubusercontent.com/16911730/268498206-06aacbb5-132d-41ae-82fc-f850f92955c9.png) | ![5c1d57e600a0159050b5f3141a511cb7](https://user-images.githubusercontent.com/16911730/268498211-2786c132-95fc-45ab-9828-6144af5c6364.png)

Exactly, I have noticed that it doesn't proc often in PvP, so my report is valid yes.

**Comment by bonfax4** [comment]
Seal of insight is using a static proc chance of 60% it seems.

Tested with a 1.0 AS weapon and the results are  : 

692/1155 = @60% chance per hit. 60% with that speed is 36 ppm which cant be accurate.

My insights healed for 135.
![8df6a2cfff05ba42c624b83ab7723e66](https://github.com/gamefreedomgit/Maelstrom/assets/16911730/7c59ca13-d408-45b5-8b5e-f85a22cfbe95)|![810a69c41152eba9c45c2b37b7247581](https://github.com/gamefreedomgit/Maelstrom/assets/16911730/c418279a-8413-4937-88e6-d6ab9f0d4361)

In comparison I should have gotten @ 25% proc chance with that weapon.

**Comment by Renehadox** [comment]
> Did a quick test with a 4.0 weapon which should give 100% chance with 15 ppm and it did not proc a couple times so smthign is def wrong.
> 
> ![5919d86e0673768f405a1ce5b1a0c271](https://user-images.githubusercontent.com/16911730/268498206-06aacbb5-132d-41ae-82fc-f850f92955c9.png) | ![5c1d57e600a0159050b5f3141a511cb7](https://user-images.githubusercontent.com/16911730/268498211-2786c132-95fc-45ab-9828-6144af5c6364.png)

From which server is that character?

I cant find you on armory...

Seal of Insight works fine from what i see, but as far as i knew it should have procced the faster your weapon is, at least it workedl ike that on other Cata servers...

**Comment by bonfax4** [comment]
It's on the ptr realm. Seals with a chance like these are/were always ppm based. It says chance on the tooltip, not 100% so clearly wherever you played it was wrong. Insight is wotlk seal of wisdom/ light combined wich were also 15 ppm back then. There's even changelogs of it still being ppm based in mop but they buffed it to 20 ppm so if you are comparing to mop privates, every weapon above 3.00 would proc it on every hit.

**Comment by Renehadox** [comment]
I heard that this was fixed, now seal of light procs even fewer times than before...

Thank you @bonfax4 for nerfing the spell...

**Comment by bonfax4** [comment]
Lmao. It's literally a buff for every weapon above 2.4 speed. What weapon are you even using ?

**Comment by Renehadox** [comment]
> Lmao. It's literally a buff for every weapon above 2.4 speed. What weapon are you even using ?

i am using 3,60 weapon 2.60 and 1.60

**Comment by Renehadox** [comment]
@bonfax4  ?

**Comment by bonfax4** [comment]
> @bonfax4 ?

?
